### PR TITLE
fix: only disable erc 1155 in sell mode

### PR DESCRIPTION
**Pull request by lynnshaoyu**
1) also fixes bug where loading state collection banner is still the old height (smaller, 160px) instead of the new 276px height. 

2) dont include erc 1155 and sus marked nft's when selecting all

### CODE REVIEW

File: src/nft/pages/collection/index.tsx [hunk 1]
```diff
@@ -113,8 +114,8 @@ const Collection = () => {
           {contractAddress ? (
             <>
               {' '}
-              <Box width="full" height="276">
-                <Box width="full" height="276">
+              <Box width="full" height={`${COLLECTION_BANNER_HEIGHT}`}>
+                <Box width="full" height={`${COLLECTION_BANNER_HEIGHT}`}>
```
Comment (cbachmeier) [inline src/nft/pages/collection/index.tsx:118]: I believe one of these divs can be removed

File: src/nft/components/profile/view/ViewMyNftsAsset.tsx [hunk 2]
```diff
@@ -93,7 +93,7 @@ export const ViewMyNftsAsset = ({
 
   const assetMediaType = Card.useAssetMediaType(asset)
 
-  const isDisabled = asset.asset_contract.tokenType === 'ERC1155' || (isSellMode && asset.susFlag)
```
Comment (cbachmeier) [inline src/nft/components/profile/view/ViewMyNftsAsset.tsx:96]: we also need to hide floor prices for 1155s not in sell mode

**Review by cbachmeier — CHANGES_REQUESTED** [review]

File: src/nft/pages/collection/index.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/nft/pages/collection/index.tsx]: u are right🙃 done

File: src/nft/components/profile/view/ViewMyNftsAsset.tsx (hunk 2 above)
Comment (lynnshaoyu) [reply src/nft/components/profile/view/ViewMyNftsAsset.tsx:96]: sg done.

File: src/nft/components/collection/Card.tsx [hunk 3]
```diff
@@ -526,6 +526,11 @@ const ProfileNftDetails = ({ asset, isSellMode }: ProfileNftDetailsProps) => {
     return !!asset.name ? asset.name : `#${asset.tokenId}`
   }
 
+  const shouldShowUserListedPrice =
+    (asset.asset_contract.tokenType === 'ERC1155' ? isSellMode && !asset.notForSale : !asset.notForSale) &&
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:530]: nit: small simplification:
`const shouldShowUserListedPrice = !!asset.floor_sell_order_price && !!asset.notForSale && (asset.asset_contract.tokenType !== 'ERC1155' || isSellMode)`

**Review by cbachmeier — CHANGES_REQUESTED** [review]
had a couple more comments on 1155 behaviour in slack

File: src/nft/components/collection/Card.tsx (hunk 3 above)
Comment (lynnshaoyu) [reply src/nft/components/collection/Card.tsx]: niceee clean done

File: src/nft/components/collection/Card.tsx [hunk 4]
```diff
@@ -527,8 +527,7 @@ const ProfileNftDetails = ({ asset, isSellMode }: ProfileNftDetailsProps) => {
   }
 
   const shouldShowUserListedPrice =
-    (asset.asset_contract.tokenType === 'ERC1155' ? isSellMode && !asset.notForSale : !asset.notForSale) &&
-    !!asset.floor_sell_order_price
+    !!asset.floor_sell_order_price && !asset.notForSale && (asset.asset_contract.tokenType !== 'ERC1155' || isSellMode)
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:530]: nit: could use TokenType.1155 here

File: src/nft/types/sell/sell.ts [hunk 5]
```diff
@@ -2,6 +2,8 @@ import { NftMarketplace, OrderStatus, OrderType } from 'graphql/data/nft/__gener
 
 import { GenieCollection, PriceInfo, TokenType } from '../common'
 
+export const ERC_1155 = 'ERC1155'
```
Comment (cbachmeier) [inline src/nft/types/sell/sell.ts:5]: is this used? looks like you're getting type from TokenType.1155

**Review by cbachmeier — APPROVED** [review]
Only small nits and a comment on select all not selecting sus assets

**Pull request merged**